source/github-events/on-field-keydown.tsx

```diff
@@ -0,0 +1,28 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+
+type DelegateFieldEvent = delegate.EventHandler<KeyboardEvent, HTMLTextAreaElement>;
+
+function onFieldKeydown(selector: string, callback: DelegateFieldEvent): void {
+	delegate<HTMLTextAreaElement, KeyboardEvent>(document, selector, 'keydown', event => {
+		const field = event.delegateTarget;
+
+		// Don't do anything if the autocomplete helper is shown or if non-latin input is being used
```

Comment:
```suggestion
		// The suggester is GitHub’s autocomplete dropdown
```

Reply:
6d852ce1d75bf9ce59bedb902300d9d5a2a8b4e6

Reply:
@fregante do you still want the wording changed?